kbase-extension/static/kbase/js/widgets/narrative_core/catalog/kbaseCatalogBrowser.js

```diff
@@ -430,81 +475,82 @@ define ([
                 } else {
                     self.clearFilter();
                 }
-
             } else {
                 self.clearFilter();
             }
 
             // hide/show sections
             const sections = self.$appListPanel.find('.catalog-section');
-            for(let i=0; i<sections.length; i++) {
+            for (let i = 0; i < sections.length; i++) {
                 $(sections[i]).show();
-                const cards = $(sections[i]).find('.kbcb-app-card-container,.kbcb-app-card-list-element');
+                const cards = $(sections[i]).find(
+                    '.kbcb-app-card-container,.kbcb-app-card-list-element'
+                );
                 let hasVisible = false;
-                for(let j=0; j<cards.length; j++) {
-                    if($(cards[j]).is(':visible')) {
+                for (let j = 0; j < cards.length; j++) {
+                    if ($(cards[j]).is(':visible')) {
                         hasVisible = true;
                         break;
                     }
                 }
-                if(!hasVisible) {
+                if (!hasVisible) {
                     $(sections[i]).hide();
                 }
             }
-
-
-
         },
 
-        clearFilter: function() {
+        clearFilter: function () {
             const self = this;
-            for(let k=0; k<self.appList.length; k++) {
+            for (let k = 0; k < self.appList.length; k++) {
                 self.appList[k].show();
             }
         },
 
-
-
-        initMainPanel: function($appListPanel, $moduleListPanel) {
-            const $mainPanel = $('<div>').addClass('container').css({'width':'100%','overflow-x':'hidden','height':'550px','overflow-y': 'auto'});
+        initMainPanel: function ($appListPanel, $moduleListPanel) {
+            const $mainPanel = $('<div>').addClass('container').css({
+                width: '100%',
+                'overflow-x': 'hidden',
+                height: '550px',
+                'overflow-y': 'auto',
+            });
```

Comment:
AST change

Reply:
Didn't have any effect - this is the widget run in the app panel slideout. Works fine. :+1: